activerecord/test/schema/schema.rb

```diff
@@ -169,7 +169,7 @@
   end
 
   create_table :comments, force: true do |t|
-    t.integer :post_id, null: false
```

Comment:
It looks like this has been around since roughly 2008 (centralized DB schema + git history gets a bit sketchy after that, looks like could have been around since 2006), but no tests break when I remove this, so... 😬